ci/infrastructure/scripts/infrastructure.sh

```
@@ -1282,4 +1301,4 @@ case $METHOD in
   ;;
 esac
 # ====/RUN ====
-exit $EXIT_CODE
+exit ${EXIT_CODE:0}
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)